cmd/openfish/handlers/user.go

```diff
@@ -35,74 +35,67 @@ LICENSE
 package handlers
 
 import (
+	"fmt"
+	"strconv"
+
 	"github.com/ausocean/openfish/cmd/openfish/api"
-	"github.com/ausocean/openfish/cmd/openfish/entities"
 	"github.com/ausocean/openfish/cmd/openfish/services"
+	"github.com/ausocean/openfish/cmd/openfish/types/role"
 
 	"github.com/gofiber/fiber/v2"
```

Comment:
Order imports, std, 3rd party, ausocean

Reply:
When I run go fmt ./cmd/openfish it reorders them into this order, so I would prefer not to change that.